airbyte-cdk/python/airbyte_cdk/sources/streams/http/error_handlers/backoff_strategy.py

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+from abc import ABC, abstractmethod
+from typing import Optional
+
+import requests
+
+
+class BackoffStrategy(ABC):
+    @property
+    @abstractmethod
+    def max_retries(self) -> int:
+        """
+        Override if needed. Specifies maximum amount of retries for backoff policy. Return None for no limit.
+        """
+        pass
+
+    @property
+    @abstractmethod
+    def max_time(self) -> int:
```

Comment:
I don't think this method needs to be part of the public interface. It seems to only be used internally by the default implementation

Reply:
Updated